### PR TITLE
Feature/add cohort client

### PR DESCRIPTION
**Pull request by dan1212111**
Screenshot 2022-04-27 at 15 10 23

Added radio buttons Student/Teacher.

### CODE REVIEW

**Review by KinTale — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ### Team Dev Client
 
-Client repository for team dev project.
-
+conflicting changes
```
Comment (Auenc) [inline README.md:3]: Is this change still needed? I would suspect you'd want to keep the original text?

File: package.json [hunk 2]
```diff
@@ -41,6 +41,6 @@
     ]
   },
   "devDependencies": {
-    "cypress": "^9.5.4"
+    "cypress": "^9.6.0"
```
Comment (Auenc) [inline package.json:44]: Was there a reason you changed the cypress version?

File: src/App.js [hunk 3]
```diff
@@ -15,6 +15,7 @@ function App() {
         <Route path="/" element={<LoginPage />} />
         <Route path="/signup" element={<RegistrationPage />} />
         <Route element={<AuthenticateUser />}>
+          <Route path="/addCohort" element={<AddCohort />} />
```
Comment (Auenc) [inline src/App.js:18]: this path might be better as `add-cohort` or maybe `cohort/add`. You want to avoid using upper case letters in the url

File: src/components/addCohort/addCohort.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+export default function AddCohort() {
+  const [cohort, setCohort] = useState("");
+
+  const handleChange = () => {
+    console.log("read event: ");
+  };
+  return (
+    <>
+      <button onclick={() => handleChange()}>Add Cohort</button>
+      {/* <Button variant="outlined">Default</Button> */}
```
Comment (Auenc) [inline src/components/addCohort/addCohort.js:14]: Is this needed anymore? If not, you can remove the commented code.

File: src/components/users/login/LoginPage.js [hunk 5]
```diff
@@ -21,11 +21,26 @@ const LoginPage = () => {
...
         localStorage.setItem(process.env.REACT_APP_USER_TOKEN, res.data.data.token)
         setLoginResponse(res.data)
+        console.log(res.data)
         navigate("../posts", { replace: true });
       })
       .catch(err => console.log(err.response))
   }
 
+
+// function getListOfCohorts() {
```
Comment (Auenc) [inline src/components/users/login/LoginPage.js:31]: Should also remove this if it is not needed anymore

File: src/components/users/registration/RegistrationPage.js [hunk 6]
```diff
@@ -5,12 +5,16 @@ import userBlankData from '../utils/userHelpers'
...
+
 const RegistrationPage = () => {
   const [user, setUser] = useState(userBlankData())
   const [registerResponse, setRegisterResponse] = useState("")
+  const [radioButtonValue, setRadioButtonValue] = useState("STUDENT")
+
 
   const registerUser = (event) => {
     event.preventDefault()
+    console.log(user)
```
Comment (Auenc) [inline src/components/users/registration/RegistrationPage.js:17]: this can be removed if not needed

**Review by Auenc — CHANGES_REQUESTED** [review]
The main changes here are to make sure you're not changing the quotes used from single to double, and to remove any commented code/redundant console.logs

File: package.json (hunk 2 above)
Comment (dan1212111) [reply package.json:45]: I installed Cypress thinking it hadn't been which must have changed the versions. I've changed them around.

File: src/App.js (hunk 3 above)
Comment (dan1212111) [reply src/App.js:16]: I've removed this path as it's no longer necessary.

File: src/components/addCohort/addCohort.js (hunk 4 above)
Comment (dan1212111) [reply src/components/addCohort/addCohort.js:14]: File removed.

File: src/components/users/login/LoginPage.js (hunk 5 above)
Comment (dan1212111) [reply src/components/users/login/LoginPage.js:41]: Done.

File: src/components/users/registration/RegistrationPage.js (hunk 6 above)
Comment (dan1212111) [reply src/components/users/registration/RegistrationPage.js:17]: Removed.

File: src/components/users/login/LoginPage.js [hunk 7]
```diff
@@ -16,17 +16,18 @@ const LoginPage = () => {
...
-      .then((res) => {
-        localStorage.setItem(process.env.REACT_APP_USER_TOKEN, res.data.data.token);
-        setLoginResponse(res.data);
-        navigate('../posts', { replace: true });
+    event.preventDefault()
+    client.post('/login', user)
+      .then(res => {
+        localStorage.setItem(process.env.REACT_APP_USER_TOKEN, res.data.data.token)
+        setLoginResponse(res.data)
+        console.log(res.data)
```
[the remark this answers is not in the document]
Comment (dan1212111) [reply src/components/users/login/LoginPage.js:24]: Sorted.

**Review by Auenc — APPROVED** [review]
Looks good to me :)

**Review by marianphillips — DISMISSED** [review]

**Review by marianphillips — APPROVED** [review]

**Review by KinTale — APPROVED** [review]
